src/components/Dashboard/TodoList.vue

```diff
@@ -532,10 +560,7 @@
 
           <!-- is this a draft correction? -->
           <template v-else-if="EnumUtilities.isStatusDraft(item) && EnumUtilities.isTypeCorrection(item)">
-            <div
-              data-test-class="correction-draft"
```

Comment:
We aren't doing anything with the data-test-class so I removed a few of them here.